.travis.yml

```diff
@@ -21,12 +21,16 @@ script:
 
 after_success:
   - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
-      export REPO=openebs/node-disk-manager-$ARCH:$TAG;
+      export REPO=openebs/node-disk-manager-$ARCH;
       docker login -u "$DNAME" -p "$DPASS";
-      docker push $REPO;
+      docker push $REPO:$TAG;
+      docker tag $REPO:$TAG $REPO:latest;
+      docker push $REPO:latest;
```

Comment:
lets have "ci" here as people have tendency to use latest to refer to latest release. Please use $REPO:ci in case of *non-release* build.

Reply:
sure @pawanpraka1 . I will change it to ci.